build.gradle

```diff
@@ -78,6 +78,7 @@ subprojects {
         compile("org.ow2.proactive:scheduling-api-graphql-beans-input-gwt:${schedulingPortalVersion}")
         compile("org.ow2.proactive:scheduling-api-graphql-common-gwt:${schedulingPortalVersion}")
         compile("org.ow2.proactive:scheduling-api-graphql-client-gwt:${schedulingPortalVersion}")
+        compile 'org.ow2.proactive:catalog-client:+'
```

Comment:
Could you replace plus with specific version of catalog client? or justify why it should be plus.

Reply:
I don't know why, I had a moment when I thought it was better to always use the latest version available in our nexus. Thank you for the warning.

Reply:
It is confusing. But in general, we use specific version. Unless in system-tests. Cause there we use latest code.